lte/gateway/c/session_manager/SessionState.h

```diff
@@ -84,7 +84,35 @@ class SessionState {
     std::vector<PolicyRule> dynamic_rules;
     std::vector<PolicyRule> gy_dynamic_rules;
     optional<AggregatedMaximumBitrate> ambr;
+    // 5G specific extensions
+    std::vector<SetGroupPDR> Pdr_rules_;
+    std::vector<SetGroupFAR> Far_rules_;
+    magma::lte::Fsm_state_FsmState state;
+    std::string sess_id;
+    uint32_t ver_no;
+    NodeId nodeId;
+    FSid Seid;
+    // 5G specific extension routines
+    magma::lte::Fsm_state_FsmState get_smf_state(struct SessionInfo);
+    // To get sess id
+    std::string get_session_id(struct SessionInfo);
```

Comment:
@gurrapualt I don't think we need the SessionInfo parameter for any of these functions. They are attached to the struct, so they should have access to the internal fields.

Reply:
Initially for clear separation it was there, now removed